nextcord/user.py

```diff
@@ -377,20 +378,25 @@ async def edit(
         .. versionchanged:: 2.1
             The ``avatar`` parameter now accepts :class:`File`, :class:`Attachment`, and :class:`Asset`.
 
+        .. versionadded:: 3.0
+            The ``banner`` field has been added.
+
         Parameters
         ----------
         username: :class:`str`
             The new username you wish to change to.
         avatar: Optional[Union[:class:`bytes`, :class:`Asset`, :class:`Attachment`, :class:`File`]]
             A :term:`py:bytes-like object`, :class:`File`, :class:`Attachment`, or :class:`Asset`
             representing the image to upload. Could be ``None`` to denote no avatar.
+        banner: Optional[Union[:class:`bytes`, :class:`Asset`, :class:`Attachment`, :class:`File`]]
+            The banner that you wish to change to. Passing ``None`` denotes no banner.
```

Comment:
I think the documentation here should be phrased more like the avatar parameter.